x/authz/client/cli/tx.go

```diff
@@ -254,17 +254,7 @@ Example:
 			if err != nil {
 				return err
 			}
-			msgs := theTx.GetMsgs()
-			serviceMsgs := make([]sdk.ServiceMsg, len(msgs))
-			for i, msg := range msgs {
-				srvMsg, ok := msg.(sdk.ServiceMsg)
-				if !ok {
-					return fmt.Errorf("tx contains %T which is not a sdk.ServiceMsg", msg)
-				}
-				serviceMsgs[i] = srvMsg
-			}
-
-			msg := types.NewMsgExecAuthorized(grantee, serviceMsgs)
+			msg := types.NewMsgExecAuthorized(grantee, theTx.GetMsgs())
 			svcMsgClientConn := &msgservice.ServiceMsgClientConn{}
```

Comment:
`ServiceMsgClientConn` can be removed too. They still work with this PR, but are a bit useless. I propose a follow-up PR (which I can take care of) to refactor that, as to keep this PR's scope minimal.